src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -68,6 +68,8 @@ export class PoolFormComponent extends CdForm implements OnInit {
   editing = false;
   isReplicated = false;
   isErasure = false;
+  blobUnits = ['KiB', 'MiB', 'GiB', 'TiB'];
+  maxBytesUnits = ['MiB', 'GiB', 'TiB'];
```

Comment:
```suggestion
  maxBytesUnits = ['KiB','MiB', 'GiB', 'TiB'];
```
Show KiB as well but make default selection GiB